packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -236,27 +236,38 @@ type SingleFormComponentProps = {
 function SingleForm({ form, appUrl, Page, enrichedWithUserProfileForm }: SingleFormComponentProps) {
   const utils = trpc.useUtils();
   const { t } = useLocale();
-
+  const isTeamForm = form.teamId;
   const [isTestPreviewOpen, setIsTestPreviewOpen] = useState(false);
   const [response, setResponse] = useState<FormResponse>({});
-  const [decidedAction, setDecidedAction] = useState<Route["action"] | null>(null);
+  const [chosenRoute, setChosenRoute] = useState<NonRouterRoute | null>(null);
```

Comment:
It was a typescript error to use Route. It should have always been NonRouteRoute. But due to some reason this TS error wasn't caught